src/models/validators/durationCoverage.js

```diff
@@ -0,0 +1,26 @@
+import { validate } from 'validate.js'
+import { validateModel } from 'models/validate'
+import { findTimelineGaps } from 'helpers/date'
+
+const durationCoverageValidator = (value, options = {}) => {
+  if (validate.isEmpty(value)) return null // Don't validate if there is no value
+
+  const { requiredDuration } = options
+  if (!requiredDuration) return 'Invalid options'
```

Comment:
These string return values seems a little odd – what the intended usage by the caller?

Reply:
(I didn't see a caller of `durationCoverageValidator` so didn't want it to hold up the PR.)

Reply:
@ryanhofdotgov So the paradigm for `validate.js` custom validators is to return `null` if the value passes validation, and an error message if it doesn't. We aren't actually using any of these error messages currently, since right now `validate.js` is only being used to update the valid state in Redux, and also because we use the error message locale files. I'm thinking that when we get to the UI refactor, we can look at whether or not to change this, and how to use these messages better (for example, maybe returning a key from the locale file instead). For now I've just been returning a string that describes what the error itself is.